### PR TITLE
fix: build with patched redis lib

### PR DESCRIPTION
**Pull request by lewxdev**
~~note: it deploys manually but not automatically, idk what that's about~~

got it, default bun version was out of date

### CODE REVIEW

File: src/api.ts [hunk 1]
```diff
@@ -1,23 +1,29 @@
...
-	return c.json({ field });
+	return c.json({ result: field });
 });
 
 api.get("/field/seed", async (c) => {
 	const redis = c.get("redis");
-	await redis.set("field", Buffer.from(_.times(5000, () => _.random(9))));
+	const bitfield = redis.bitfield("field");
+	await redis.del("field");
+	for (const [index, value] of new Field().data.entries()) {
```
Comment (drewradcliff) [inline src/api.ts:17]: It would be better to format the data into a string or buffer and then set with a single query. Unless upstash works differently.

**Review by drewradcliff — COMMENTED** [review]
Nice work! Although the build keeps failing in CF after I updated the env variables.

**Comment by lewxdev** [comment]
> Nice work! Although the build keeps failing in CF after I updated the env variables.

I have a fix for this [via the Cloudflare Discord](https://discord.com/channels/595317990191398933/1134543567004631121/1152321372786274314) ([source](https://gist.github.com/Hebilicious/88e5a444f42b8dc09fb86dfa865c6ed3?permalink_comment_id=4722095#gistcomment-4722095)):

> Just like in other tools, you can specify exact versions (e.g. by setting the env var `BUN_VERSION` to `1.0.2`) The Pages build image ships with bun 1.0.1. We'll update pre-installed versions periodically

we need to set the bun version to latest, there's a fix for the rollup build also see [pi0/bun-on-pages](https://github.com/pi0/bun-on-pages)

File: src/api.ts (hunk 1 above)
Comment (lewxdev) [reply src/api.ts:17]: this works as expected, (I've tested it and it sends a single query with the bitfield command I created)
but I have some ideas for the architecture that may upend this I want to run by you

**Comment by drewradcliff** [comment]
> I have a fix for this [via the Cloudflare Discord](https://discord.com/channels/595317990191398933/1134543567004631121/1152321372786274314) ([source](https://gist.github.com/Hebilicious/88e5a444f42b8dc09fb86dfa865c6ed3?permalink_comment_id=4722095#gistcomment-4722095)):

"We'll update pre-installed versions periodically"
image

**Review by drewradcliff — APPROVED** [review]

**Pull request merged**